src/compas/geometry/primitives/frame.py

```diff
@@ -645,29 +690,29 @@ def to_local_coordinates(self, object_in_wcf):
         >>> pl = frame.to_local_coordinates(pw) # point in frame
         >>> frame.to_world_coordinates(pl)
         Point(2.000, 2.000, 2.000)
+
         """
         T = Transformation.from_change_of_basis(Frame.worldXY(), self)
-        if isinstance(object_in_wcf, list):
-            return Point(*object_in_wcf).transformed(T)
-        else:
-            return object_in_wcf.transformed(T)
+        if isinstance(obj, (list, tuple)):
+            return Point(*obj).transformed(T)
+        return obj.transformed(T)
 
-    def to_world_coordinates(self, object_in_lcf):
+    def to_world_coordinates(self, obj):
```

Comment:
i would tend to keep the `lcf` (or `wcf` ) references in param names, sometimes this thing is very confusing and it is not bad to have it explicit in the name.